schwab_api/schwab.py

```diff
@@ -298,12 +299,14 @@ def trade_v2(self,
 
         response = json.loads(r.text)
 
-        messages = list()
+        # messages = list()        
+        messages.append('## place order message ##')
         if "orderMessages" in response["orderStrategy"] and response["orderStrategy"]["orderMessages"] is not None:
             for message in response["orderStrategy"]["orderMessages"]:
                 messages.append(message["message"])
 
         if response["orderStrategy"]["orderReturnCode"] in valid_return_codes:
+            messages.append('fail reason: orderReturnCode({}) is not valid_return_codes{}'.format(response["orderStrategy"]["orderReturnCode"], str(valid_return_codes)))
```

Comment:
Is this message in the right place? It looks like it's mentioning a fail reason even when the response **is** in the set of valid return codes

Reply:
sorry, copy & paste error. The mistaken has been correct.